viewer/packages/sector-parser/tests/GltfSectorParser.test.ts

```diff
@@ -187,4 +197,35 @@ describe(GltfSectorParser.name, () => {
     // Quad geometry
     expect(trapeziums.attributes['position'].count).toBe(4);
   });
+
+  test('Parsing draco encoded sector should return proper triangle mesh', () => {
+    const triangleMeshes = parsedDracoResult.filter(x => x.type === RevealGeometryCollectionType.TriangleMesh);
+    expect(triangleMeshes.length).toBe(1);
+
+    const { geometryBuffer } = triangleMeshes[0];
+
+    const attributes = geometryBuffer.attributes;
+
+    const attributeNames = Object.keys(attributes);
+
+    expect(attributeNames.length).toBe(3);
+
+    expect(attributeNames.includes('position')).toBeTrue();
+    expect(attributeNames.includes('treeIndex')).toBeTrue();
+    expect(attributeNames.includes('color')).toBeTrue();
+
+    // Test if all attributes point to the same underlying buffer
+    const underlyingBuffers = Object.values(attributes).map(p => (p.array as TypedArray).buffer);
+    const bufferInstance = underlyingBuffers[0];
+
+    for (let i = 1; i < underlyingBuffers.length; i++) {
+      const buffer = underlyingBuffers[i];
+      expect(buffer).toEqual(bufferInstance);
+    }
+
+    const triangleMeshNumberOfVertices = 13751;
+    expect(attributes.position.count).toBe(triangleMeshNumberOfVertices);
+    expect(attributes.treeIndex.count).toBe(triangleMeshNumberOfVertices);
+    expect(attributes.color.count).toBe(triangleMeshNumberOfVertices);
+  });
 });
```

Comment:
It would be nice to have another test checking that the results from Draco and non-Draco input is the same for the same original file.